homeassistant/components/wemo.py

```diff
@@ -123,32 +125,37 @@ def setup_url_for_address(host, port):
 
     devices = []
 
-    for host, port in config.get(DOMAIN, {}).get(CONF_STATIC, []):
-        url = setup_url_for_address(host, port)
+    if config.get(DOMAIN, {}).get(CONF_STATIC, []):
```

Comment:
Store the list of tuples the first time we do the look up.

Reply:
Why do we need this check? Why not just iterate the list?

Reply:
I was doing the check mainly so I wouldn't log the message about scanning the statically configured devices if none had been configured. I was also worried about the for loop breaking if the option configuration entry it's looking for had been omitted. I'm ok with taking this out.